tgmc.dme

```diff
@@ -1350,12 +1214,55 @@
 #include "code\modules\surgery\implant.dm"
 #include "code\modules\surgery\internal_bleeding.dm"
 #include "code\modules\surgery\necrosis.dm"
+#include "code\modules\surgery\operatingcomputer.dm"
+#include "code\modules\surgery\optable.dm"
 #include "code\modules\surgery\organs_internal.dm"
 #include "code\modules\surgery\robolimbs.dm"
+#include "code\modules\surgery\sleeper.dm"
 #include "code\modules\surgery\surgery.dm"
+#include "code\modules\surgery\surgery_tools.dm"
+#include "code\modules\telecomms\broadcasting.dm"
+#include "code\modules\telecomms\communications.dm"
+#include "code\modules\telecomms\machine_interactions.dm"
+#include "code\modules\telecomms\telecomunications.dm"
+#include "code\modules\telecomms\computers\logbrowser.dm"
+#include "code\modules\telecomms\computers\telemonitor.dm"
+#include "code\modules\telecomms\machines\allinone.dm"
+#include "code\modules\telecomms\machines\broadcaster.dm"
+#include "code\modules\telecomms\machines\bus.dm"
+#include "code\modules\telecomms\machines\hub.dm"
+#include "code\modules\telecomms\machines\processor.dm"
+#include "code\modules\telecomms\machines\receiver.dm"
+#include "code\modules\telecomms\machines\relay.dm"
+#include "code\modules\telecomms\machines\server.dm"
+#include "code\modules\teleportation\teleportation.dm"
+#include "code\modules\teleportation\teleporter.dm"
 #include "code\modules\tgs\event_handler.dm"
 #include "code\modules\tgs\includes.dm"
 #include "code\modules\tooltip\tooltip.dm"
+#include "code\modules\toys\ashtray.dm"

```

Comment:
Toys are just a kind of item. They should be in something like `code\modules\item\toys\` or `code\modules\obj\item\toys\`

Reply:
https://discordapp.com/channels/498569646014857217/498576390753615882/595256059921170451